css-fonts/styles.css

```diff
@@ -0,0 +1,13 @@
+body {
+font-family: 'Anton', sans-serif;
+font-family: 'Benne', serif;
+font-family: 'Rajdhani', sans-serif;
+font-family: 'Signika', sans-serif;
+ line-height: 1.5em;
+}
+
+h1,h2 {
+ font-family: 'Anton', sans-serif;
+ font-family: 'Benne', serif;
```

Comment:
You can't set multiple font families on one element like this. A single property can only be declared once in a single CSS rule. You can choose one of these and apply that one.